Utils/Book.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Assignment_1.Utils
+{
+    /// <summary>
+    /// Represents a book in the library system.
+    /// </summary>
+    internal class Book
```

Comment:
What about the idea of having multiple copies of same book? You could have created a parent Book class with common details and a BookItem class defining a specific copy of book.